### ISSUE TITLE
[v4.0.beta12.mono.official] blend files not imported correctly: missing textures, weird white light

### ISSUE THREAD

**Issue by blender-girl** [godotengine/godot#71371]
### Godot version

v4.0.beta12.mono.official [3c9bf4b]

### System information

Windows 10 Home 64 bit, NVIDIA GeForce GTX 1070, 31.0.15.1748 (NVIDIA 517.48), forward_plus

### Issue description

When Importing a .blend file, it's displayed without any textures and the model is giving an incredible big white light that affects all models in the scene.

See screenshot: model at the left is a .blend file, model 2 and 3 are GLTF files.
The .blend file is white, no textures and puts a white light on model 2 and 3 which causes parts of those perfectly fine models to become white too. The blend model is displayed correctly in Blender, with all textures.

Scene with blend model at the left:

![afbeelding](https://user-images.githubusercontent.com/122493908/212440275-1470414f-90e8-48da-9a43-7c8b48d33a77.png)

Scene without the blend model at the left (the white light is gone and the other 2 models are back to normal state when you remove the blend model)

![afbeelding](https://user-images.githubusercontent.com/122493908/212440316-7cebd609-4b20-4d4f-a14e-e82c16db93a3.png)

### Steps to reproduce

- install Godot v4.0.beta12.mono.official [3c9bf4b] on Windows 10
- create empty project (forward_plus)
- set path to Blender 3.4.1 executable
- import 2 GLTF models
- import blend model
- see how the blend model affects the whole scene with a white light, also making parts of other models white

The models are included in attachment.
The blend model is perfectly fine in Blender

[models.zip](https://github.com/godotengine/godot/files/10416169/models.zip)

### Minimal reproduction project

N/A

**Comment by blender-girl** [comment]
Additional question: does Godot require you to pack resources into your blend file?
I did not do this, so all textures are in the same folder as the blend file.

![afbeelding](https://user-images.githubusercontent.com/122493908/212441024-58b82ec5-307c-4b39-ae36-170a08c03cfa.png)

**Comment by dc1248** [comment]
It all becomes clear once you know that Godot's Blender importer simply runs Blender in batch mode and exports a GLTF. I haven't been able to test it so I don't know if offers any control over GLTF export options, but I'm guessing it uses the wrong options.

Also, Godot imports GLTF lights with the wrong range+energy (like a million times too bright) and shadows off. I don't think that's your issue but it's something to be aware of. (I hope someone with more experience picks up on this and creates a fix or bug report... if not maybe I will eventually.)

And no, it shouldn't matter if you pack resources into your blend file.

**Comment by blender-girl** [comment]
Yes, the light issue I have too! See screenshot. It's like a lightning bolt is inside your scene and all other objects become white...

This bug is still present in beta 13.

**Comment by lmsonic** [comment]
The issues is that the yard model has a light inside, so it blows up the scene white
![image](https://github.com/godotengine/godot/assets/25965198/02efbd52-2562-4cd1-a4b9-8697a9e3eff8)
![image](https://github.com/godotengine/godot/assets/25965198/d3e24ac0-dc2c-4610-ab7a-0ca91e82478c)
